packages/ui/src/components/Renderer.tsx

```diff
@@ -54,6 +55,13 @@ const Renderer = (props: RendererProps) => {
   const ref = useRef<HTMLDivElement>(null);
   const _cache = useRef<Map<any, any>>(new Map());
 
+  const flatSchemaWithoutData = JSON.stringify(schema, (key, value) => {
```

Comment:
this was an important change to allow the propPanel to be updated when making changes within the content editable text box. 

without this it will rerender the schema on each change of these values, which moves the cursor to the end of the content editable (as well as being janky).